docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md

```diff
@@ -78,6 +78,8 @@ python3 mo.py --input_model wsj_dnn5b_smbr.nnet --counts wsj_dnn5b_smbr.counts -
 	\f$|C|\f$ - number of elements in the counts array;
   * The normalized counts are subtracted from biases of the last or next to last layer (if last layer is SoftMax).
 
+  > **NOTE:** Model Optimizer will show warning if model contains counts values inside model. But counts values will not be applyed if `--counts` option is not used.
```

Comment:
I propose to simplify this sentence as follows:
Model Optimizer will show warning if model contains counts values inside model and `--counts` option is not used.